app/pb/communities.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package org.couchers.api.communities;
+
+import "google/protobuf/timestamp.proto";
+
+import "pb/api.proto";
+
+message Point {
+  double lat = 1;
+  double lng = 2;
+}
+
+message Node {
+  int32 node_id = 1;
+  string name = 2;
+  org.couchers.api.core.NullableInt32Value parent_node = 3;
```

Comment:
I don't think this needs to be nullable, no parent could just be 0 since autoincrementing IDs start from 1 (right?)